packages/core/src/node/backend-connection-status.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as express from 'express';
+import { injectable } from 'inversify';
+import { BackendApplicationContribution } from './backend-application';
+
+@injectable()
+export class BackendConnectionStatusEndpoint implements BackendApplicationContribution {
+
+    protected app: express.Application | undefined;
+
+    configure(app: express.Application): void {
+        this.app = app;
+    }
+
+    onStart(): void {
+        this.app!.get('/alive', (request, response) => response.send());
```

Comment:
I meant `configure` is callback which should be used to install handlers, like: 
```ts
configure(app: express.Application): void {
    app.get('/alive', (request, response) => response.send());
}
```

Reply:
I see, of course. Thank you!